fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt

```diff
@@ -22,8 +24,15 @@ import org.mozilla.gecko.search.SearchWidgetProvider
 
 class SearchEngineFragment : PreferenceFragmentCompat() {
 
+    private var newUI: Boolean = false
```

Comment:
```suggestion
    private val shouldUseNewUI: Boolean by lazy {
        requireContext().settings().enableNewSearchSettings
    }
```

Reply:
It breaks `SearchEngineFragmentTest`, as requireContext() would be null. And with the way the test is written I don't think we make it work.